test/images/image-util.sh

```diff
@@ -21,63 +21,63 @@ set -o pipefail
 TASK=$1
 IMAGE=$2
 
-KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE}")/../.." && pwd -P)"
+KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
 source "${KUBE_ROOT}/hack/lib/util.sh"
 
 # Mapping of go ARCH to actual architectures shipped part of multiarch/qemu-user-static project
 declare -A QEMUARCHS=( ["amd64"]="x86_64" ["arm"]="arm" ["arm64"]="aarch64" ["ppc64le"]="ppc64le" ["s390x"]="s390x" )
 
 # Returns list of all supported architectures from BASEIMAGE file
 listArchs() {
-  cut -d "=" -f 1 ${IMAGE}/BASEIMAGE
+  cut -d "=" -f 1 "${IMAGE}/BASEIMAGE" | (mapfile -t; echo "${MAPFILE[@]}")
```

Comment:
There was intermixed use of arrays and strings. Adding the mapfile and returning an array normalizes on the use of arrays throughout the script.